projects/igniteui-angular/src/lib/grids/summaries/grid-summary.service.ts

```diff
@@ -116,16 +116,38 @@ export class IgxGridSummaryService {
             rowSummaries = new Map<string, IgxSummaryResult[]>();
             this.summaryCacheMap.set(rowID, rowSummaries);
         }
+
```

Comment:
in order to fix the issue with the height of the summary row consider adding the following logic into `calcMaxSummaryHeight` method on row 101
```suggestion
            const getCurrentSummary = column.summaries.operate([], [], column.field);
            const getCurrentSummaryColumn = column.disabledSummaries.length > 0 ?
            getCurrentSummary.filter(s => !column.disabledSummaries.includes(s.key)).length :
            getCurrentSummary.length;
```